homeassistant/components/advantage_air/entity.py

```diff
@@ -62,6 +62,12 @@ def __init__(self, instance: AdvantageAirData, ac_key: str) -> None:
     def _ac(self) -> dict[str, Any]:
         return self.coordinator.data["aircons"][self.ac_key]["info"]
 
+    @property
+    def _myzone(self) -> dict[str, Any]:
+        return self.coordinator.data["aircons"][self.ac_key]["zones"].get(
```

Comment:
Either the type annotation or the code is wrong. `dict.get` will return `None` if the key is missing in the dict.

Reply:
Sorry, I did try set this and then had some odd issue locally and it got missed.
PR to add ` | None` https://github.com/home-assistant/core/pull/101377